### PR TITLE
Added the best resources for learning Blockchain with hands-on experience 

### LINKED ISSUE #561: Want to add some more resource for Blockchain and Defi
### Description

I believe that there are some really good sources for learning Blockchain and Defi. So I want to add it so that others can use it too.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Parthmandale**
## Fixes Issue

Closes #561 
## Changes proposed

Added three resources for Blockchain and one for Defi
## Screenshots

ROADMAPS

### CODE REVIEW

**Comment by Parthmandale** [comment]
@siddhant-khisty Thanks for assigning me this issue, I've completed it. I hope this will help many blockchain enthusiasts folks.

**Comment by AnuragThePathak** [comment]
@Parthmandale ignore the workflow, ig there's something wrong with it.

File: Blockchain/readme.md [hunk 1]
```diff
@@ -17,6 +17,10 @@
 - [What iS A DAO](https://www.forbes.com/sites/cathyhackl/2021/06/01/what-are-daos-and-why-you-should-pay-attention/?sh=343b04067305)
 - [Intro To DAPPS](https://ethereum.org/en/developers/docs/dapps/)
 - [EVM and Non-EVM Chains](https://ethereum.org/en/developers/docs/dapps/)
+- [Metaschool: Free cources](https://metaschool.so/courses)
+- [Lumos Academy: Learning Platform](https://academy.lumoslabs.co/courses)
+- [DeFi Infrastructure - Youtube](https://www.youtube.com/playlist?list=PLE1Vu6ctbqa61FqJmPDnoPzDqrnyILHRm)
+- [Learn by reading Blogs](https://parthmandale.hashnode.dev/series/blockchain-defi)
```
Comment (siddhant-khisty) [inline Blockchain/readme.md:23]: These seem to be a complete learning roadmap for blockchain. It would be better to make a separate section at the end of the roadmap called `Additional resources` and link these in there

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

File: Blockchain/readme.md (hunk 1 above)
Comment (Parthmandale) [reply Blockchain/readme.md:23]: Alright @siddhant-khisty ser, I'm adding `Additional resources for Blockchain Fundamentals` next to it.

**Comment by Parthmandale** [comment]
## Fixes Issue
Closes #561 

## Changes proposed
@siddhant-khisty I've added a separate section for additional resources as you said. Now you can close this as a completed issue!

## Screenshot 
resources

File: Blockchain/readme.md [hunk 2]
```diff
@@ -21,6 +21,12 @@
 - [Road to Web3](https://www.web3.university/tracks/road-to-web3)
 - [One of the best Web3 Roadmap](https://vitto.cc/web3-and-solidity-smart-contracts-development-roadmap/)
 
+### Additional resources for Blockchain Fundamentals
+- [Metaschool: Free cources](https://metaschool.so/courses)
+- [Lumos Academy: Learning Platform](https://academy.lumoslabs.co/courses)
+- [DeFi Infrastructure(Youtube)](https://www.youtube.com/playlist?list=PLE1Vu6ctbqa61FqJmPDnoPzDqrnyILHRm)
+- [Learn by reading Blogs](https://parthmandale.hashnode.dev/series/blockchain-defi)
```
Comment (siddhant-khisty) [inline Blockchain/readme.md:28]:
```suggestion
```
We only want to include credible resources.

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

File: Blockchain/readme.md (hunk 2 above)
Comment (Parthmandale) [reply Blockchain/readme.md:28]: Sir trust me those blog series are awesome, more blogs are going to be added to it. So it will be a beginner to advance series and anyone can learn from it in proper sequence. 
@siddhant-khisty 🙏 please let that be in additional resources.

Comment (Parthmandale) [reply Blockchain/readme.md:28]: @siddhant-khisty Please merge my PR!

Comment (Parthmandale) [reply Blockchain/readme.md:28]: will you please respond?

Comment (Parthmandale) [reply Blockchain/readme.md:28]: @siddhant-khisty Im confused about what to do please suggest!

**Comment by Parthmandale** [comment]
@siddhant-khisty will u pls check it now!

**Comment by AnuragThePathak** [comment]
@Parthmandale fix the conflicts.

File: Blockchain/readme.md [hunk 3]
```diff
@@ -115,4 +120,4 @@
   - [Austin Griffith](https://www.youtube.com/channel/UC_HI2i2peo1A-STdG22GFsA)
   - [Nader Dabit](https://www.youtube.com/user/boyindasouth)
   - [Harkirat Singh (Youtube)](https://www.youtube.com/playlist?list=PLVKLWop9wWA82pZoyylZD2VF2c7MR8_5I)
-  - [Daulat Hussain (Youtube)](https://www.youtube.com/@daulathussain/featured)
```
Comment (AnuragThePathak) [inline Blockchain/readme.md:118]: Would you like to explain why did you remove this YT link?

Comment (Parthmandale) [reply Blockchain/readme.md:118]: That line was coming in conflicts! sorry if I made any mistake. I just want to add the resources I have mentioned. please tell me what are conflicts coming?

Comment (Parthmandale) [reply Blockchain/readme.md:118]: So should I close this PR and open a new one?

Comment (AnuragThePathak) [reply Blockchain/readme.md:118]: Your PR, so whatever way you are comfortable with.
